### DISCUSSION TITLE
pygaD- what's the D?

### DISCUSSION THREAD

**Discussion by jdmoore7** [ahmedfgad/GeneticAlgorithmPython#135] [General]
Where does the D come from? It's not in 'python', 'genetic' or 'algorithms'

**Comment by ahmedfgad** [comment]
This is mentioned in the [PyGAD: An Intuitive Genetic Algorithm Python Library](https://arxiv.org/abs/2106.06158) paper.

> The name PyGAD has 3 parts:
> 1) Py which means it is a Python library. This is a common naming convention for Python libraries.
> 2) GA stands for genetic algorithm.
> 3) D for decimal because the library was originally supporting only decimal genetic algorithm. Now, it supports both decimal and binary genetic algorithm.

You might think that PyGA is enough but there is an existing library with that name already at PyPI. This is why the extra `D` was added.
